templates/js/custom-copilot-rag-custom-api/README.md.tpl

```diff
@@ -34,31 +33,30 @@ The app template is built using the Teams AI library, which provides the capabil
 1. In file *env/.env.testtool.user*, fill in your OpenAI key `SECRET_OPENAI_API_KEY=<your-key>`.
 {{/useOpenAI}}
 {{#useAzureOpenAI}}
-1. In file *env/.env.testtool.user*, fill in your Azure OpenAI key `SECRET_AZURE_OPENAI_ENDPOINT=<your-key>` and endpoint `SECRET_AZURE_OPENAI_ENDPOINT=<your-endpoint>`.
-1. In `src/app/app.js`, update `azureDefaultDeployment` to your own model deployment name.
+1. In file *env/.env.testtool.user*, fill in your Azure OpenAI key `SECRET_AZURE_OPENAI_ENDPOINT=<your-key>`, endpoint `SECRET_AZURE_OPENAI_ENDPOINT=<your-endpoint>` and deployment name `AZURE_OPENAI_DEPLOYMENT=<your-deployment-name>`.
 {{/useAzureOpenAI}}
 1. Press F5 to start debugging which launches your app in Teams App Test Tool using a web browser. Select `Debug in Test Tool (Preview)`.
 1. You can send any message to get a response from the bot.
 
 **Congratulations**! You are running an application that can now interact with users in Teams App Test Tool:
 
-![ai chat bot](https://github.com/OfficeDev/TeamsFx/assets/9698542/9bd22201-8fda-4252-a0b3-79531c963e5e)
+![custom api template](https://github.com/OfficeDev/TeamsFx/assets/63089166/c5a8c749-9b96-4d1e-89c9-576621a9bc20)
 {{/enableTestToolByDefault}}
```

Comment:
Seems like the image shown in the screenshot is broken?

![image](https://github.com/OfficeDev/TeamsFx/assets/11220663/5f445b06-a3ac-4773-a05c-6ccc97914b77)

Reply:
Image here is because we are using pet store api from open api v3. It is just a sample api and no real image here. It returns url like "url1".